wix/Lang/OpenJDK.Base.en-us.wxl.template

```diff
@@ -17,7 +17,7 @@
 	<String Id="FeatureJarFileRunWithDescription">Associate .jar files to run with {vendor_branding}</String>
 	<String Id="DowngradeDenied">"A later version of [ProductName] is already installed. Setup will now exit."</String>
 	<String Id="FeatureOracleJavaSoftName">JavaSoft (Oracle) registry keys</String>
-	<String Id="FeatureOracleJavaSoftDescription">Overwrites the reg keys HKLM\Software\JavaSoft (Oracle). After uninstallation of {vendor_branding}, Oracle Java running from PATH "C:\Program Files (x86)\Common Files\Oracle\Java\javapath" does not work. You need to reinstall it to recreate these registry keys</String>
+	<String Id="FeatureOracleJavaSoftDescription">Overwrites the reg keys HKLM\Software\JavaSoft (Oracle). After uninstallation of Temurin, Oracle Java needs to be reinstalled to re-create these registry keys.</String>
```

Comment:
I think you need to keep {vendor_branding} for other folks who use these scripts (MSFT and IBM for example)

Reply:
If we keep {vendor_branding}, we need to make the string even smaller.

Reply:
I have updated the text
![image](https://github.com/adoptium/installer/assets/13155063/fc2f0eb3-9ce8-4a25-890c-9de0d1a68071)